api/package.json

```diff
@@ -7,7 +7,9 @@
   "scripts": {
     "start": "webpack && node ./dist/bundle.js",
     "develop": "webpack --watch",
-    "lint": "eslint ./src --fix"
+    "lint": "eslint src --ext .js,.vue",
```

Comment:
I don't think we need `,.vue` here since there shouldn't be any vue files in the `api`. Not a big deal either way though.

Reply:
True! It's removed in 4aac31465742e3d0053baa1e5d6304d653e508fb